### PR TITLE
Add hero banner

### PR DESCRIPTION
**Pull request by jugglingdev**
This pull request includes the hero banner.

The image dimensions match the dimensions listed in the wireframe. 

Please note that the hero is not currently responsive.  It is currently formatted for a L laptop (1440px width).

### CODE REVIEW

**Review by ZackBradshaw — COMMENTED** [review]
looks good however try to move away from using hard-coded pixel values instead than use REM units our system is set up to so that 1 rem is 10px

File: templates/_modules/hero.twig
Comment (ZackBradshaw) [inline templates/_modules/hero.twig:1]: Make sure to add alt descriptions of images this helps with SEO and accessibility for anyone using a screen reader

**Comment by jugglingdev** [comment]
Levi said to make the hero banner responsive at a laptop size and greater.  I slightly adjusted the padding around the text so that it doesn't get funky as the screen size grows (it previously made the left div grow taller than the photos on the right; now it remains the same height).

This will better suit all desktop screen sizes.

image

**Review by ZackBradshaw — COMMENTED** [review]
Looks good I like the separation of related styles. Height and width may need to be switched to something more responsive for mobile design  For example: going off device height or a percentage of parent element. As far as images go I didn't realize that those were placeholders so leaving alt out was totally ok mb :). but still its a good practice to get into

**Comment by jugglingdev** [comment]
I updated the outer hero container to display: flex like we talked about on Tuesday.  That should make it easier for mobile responsiveness in future Issues.

**Comment by ZackBradshaw** [comment]
@glitchedmob I've looked this over and It looks good to me. We should work towards merging these this Tuesday I'd like to have a more formal code and demo night once we get these all merged and main updated so that we can see we're we are at with the website as a whole

**Review by ZackBradshaw — APPROVED** [review]

**Pull request merged**